bash/compile_python_packages.sh

```diff
@@ -18,3 +18,5 @@ rm --force ${path_to_requirements}/requirements.txt
 echo -e "🛠 compiling from ${path_to_requirements}/requirements.in"
 CUSTOM_COMPILE_COMMAND="./${RELATIVE_SCRIPTPATH} ${path_to_requirements}" python3 -m piptools compile --output-file=${path_to_requirements}/requirements.txt ${path_to_requirements}/requirements.in
 echo -e "✅ done compiling ${path_to_requirements}/requirements.txt"
+
+sed -e 's/\[[^][]*\]//g' $path_to_requirements/requirements.txt > constraints.txt
```

Comment:
This converts cases like `coverage[toml]==7.3.1` to `coverage==7.3.1`. The former is a valid requirement, pinnin coverage to 7.3.1 and installing option `toml` of the package, but it's not valid in a constraints file used in pip install